### PR TITLE
Add in transitions and move deps to dev-deps in package

### PR DESCRIPTION
**Pull request by samjbmason**
Hey all 
So I have added in a transition to the page when it updates to a new color combo, I have also had to remove all of the eslint stuff as it was blocking me from installing on my system due to plugins needing specific versions of eslint. By all means feel free to get me to sort it out!

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default function configureStore (initialState) {
 
   const store = createStoreWithMiddleware(rootReducer, initialState)
 
-  if (module.hot) {
+  if (module.hot && process.env.NODE_ENV !== 'production') {
```
Comment (johnotander) [inline src/store/index.js]: Whoops. Thanks!

**Comment by johnotander** [comment]
🎉 🎉 🎉 🎉 

Regarding the eslint changes, what errors in particular where you getting? It would be awesome to get it sorted out so that it is working on your end. But I'm totally cool re-adding it back after merging in your PR. After all, functionality is more important than linting 😎 🌴 ☀️ .

**Comment by johnotander** [comment]
Actually, as a result of #42 I'll go ahead and merge this since @lachlanjc is working on improving our linting anyway.

Thanks again for the contribution, this is great!

File: src/containers/App.js [hunk 2]
```diff
@@ -130,13 +131,18 @@ const App = React.createClass({
             <strong>{contrast.score(colorOne, colorTwo)}</strong> / <strong>{contrast.ratio(colorOne, colorTwo).toFixed(2)}</strong>
           </p>
         </div>
-        <Combination colorOne={colorOne} colorTwo={colorTwo} />
+        <ReactCSSTransitionGroup
+          transitionName="color-switch"
+          transitionEnterTimeout={1300}
```
Comment (lachlanjc) [inline src/containers/App.js]: Could we lower these durations? I feel like I'm waiting for a huge process to occur while the fading happens :neutral_face: 

256, perhaps?

Comment (samjbmason) [inline src/containers/App.js]: Im totally happy to lower if everyone else wants that. My reasoning was that it gave enough time between colour combos to "cleanse the palette".

**Comment by samjbmason** [comment]
Awesome work on implementing Standard Im a big fan. If you would still like the errors Im totally happy to put it back in and send you the errors occurring @johnotander ?

File: src/containers/App.js (hunk 2 above)
Comment (lachlanjc) [inline src/containers/App.js]: It's true, but I think waiting = less voting, and it makes the site feel slow overall

Comment (johnotander) [inline src/containers/App.js]: Yeah, I definitely like the "palette cleansing" but it does now feel a touch slow. Perhaps split the difference and do ~750 so it's a little shorter but long enough to cleanse.

Comment (samjbmason) [inline src/containers/App.js]: Hopefully resolved in #45

**Pull request merged**